Tests/SentryTests/SentryClientTests.swift

```diff
@@ -477,6 +486,36 @@ class SentryClientTest: XCTestCase {
         }
     }
     
+    func testEnvironmentIsSetInEventTakesPrecedenceOverOptions() {
+        let optionsEnvironment = "environment"
+        let event = Event()
+        event.environment = "event"
+        let scope = Scope()
+        scope.setEnvironment("scope")
+        let eventId = fixture.getSut(configureOptions: { options in
+            options.environment = optionsEnvironment
+        }).capture(event: event, scope: scope)
+
+        XCTAssertNotNil(eventId)
```

Comment:
`l`: I think it is better to check that it is not empty instead of nil. That applied to all `XCTAssertNotNil(eventId)` in this PR.
```suggestion
        eventId.assertIsNotEmpty()
```

Reply:
Fixed